frontends/web/src/routes/account/send/utxos.module.css

```diff
@@ -38,9 +39,6 @@
 }
 
 .utxoExplorer {
-    background: none;
-    border: none;
-    cursor: pointer;
     margin-top: calc(var(--space-quarter) * -1);
     padding: var(--space-quarter);
 }
```

Comment:
![Screen Shot 2023-07-27 at 09 47 44](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/29a11149-d3ee-41ed-a39b-12b967c0cac7)

(with focus on open-in-explorer anchor)